jfleet-core/src/test/java/org/jfleet/util/DatabaseContainers.java

```diff
@@ -0,0 +1,65 @@
+package org.jfleet.util;
+
+import static org.jfleet.parameterized.Databases.JdbcMySql;
+import static org.jfleet.parameterized.Databases.JdbcPosgres;
+import static org.jfleet.parameterized.Databases.MySql;
+import static org.jfleet.parameterized.Databases.Postgres;
+import static org.jfleet.parameterized.IsMySql5Condition.isMySql5Present;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.jfleet.parameterized.Databases;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.JdbcDatabaseContainer;
+import org.testcontainers.containers.MySQLContainer;
+import org.testcontainers.containers.PostgreSQLContainer;
+
+public class DatabaseContainers {
+
+    private static final String MYSQL_5_VERSION = "mysql:5.7.34";
+    private static final String MYSQL_8_VERSION = "mysql:8.0.39";
+    private static final String POSTGRES_12_VERSION = "postgres:12-alpine";
+
+    private static final Map<Databases, JdbcDatabaseContainer<?>> map = new HashMap<>();
+    static {
+        map.put(Postgres, createPostgresContainer());
```

Comment:
Launching tests I see that two postgres and two mysql instances are launched, but we need just one type of each type.

Change this code to instantiate one MySql Container and one Postgres Container

Reply:
What about the MySQL version 5 and 8? Don't we need them to try the 2 connections?

Reply:
No, because running tests you select which database to test with the launch parameter.

In an execution you test v5 or v8, but not both in the same run.